DOL.WHD.Section14c.Web/src/modules/components/userRegistrationForm/userRegistrationFormTemplate.html

```diff
@@ -26,29 +23,13 @@ <h2 class="usa-alert-heading">Account Created</h2>
   </div>
   <div class="createAcct">
     <h2>Create an Account</h2>
-    <div ng-class="vm.einError || vm.einRequired || vm.invalidEin ? 'usa-input-error' : ''">
-      <label for="EIN">Employer Identification Number (EIN)</label>
-      <span class="usa-input-error-message" id="input-error-message" role="alert" ng-show="vm.einError || vm.einRequired || vm.invalidEin">
-        <div ng-show="vm.einError">
-          <p>The EIN you provided is already registerd. Please contact the account admin for access.</p>
-          <p>If you do not know the account admin, contact our office at 000-123-4567</p>
-        </div>
-        <div ng-show="vm.einRequired">
-          <p>EIN is required.</p>
-        </div>
-        <div ng-show="vm.invalidEin">
-          <p>Invalid EIN format. Example XX-XXXXXXX</p>
-        </div>
-      </span>
-      <input class="ein sidelabeled" name="EIN" id="EIN" type="text" mask="99-9999999" ng-model="formVals.ein" />
-      <span class="example">Example: XX-XXXXXXX</span>
-      <a class="dol-additional-info" href ng-click="vm.toggleEinHelp()">Why do you need an EIN to create an account?</a>
+    <div ng-class="vm.emailAddressError || vm.emailAddressRequired ? 'usa-input-error' : ''">
```

Comment:
I'm guessing this `ng-class` is not right since it refers to an email address error, but the form element is the first name.